examples/complete/fixtures.secure.tfvars

```diff
@@ -5,5 +5,6 @@ enable_self_managed_nodegroups = true
 bastion_tenancy                = "dedicated"
 eks_worker_tenancy             = "dedicated"
 cluster_endpoint_public_access = false
+eks_use_mfa                    = false
```

Comment:
Again might not be following intent here, should this be true for the secure tfvars?